package.json

```diff
@@ -49,6 +50,6 @@
     "@types/react-dom": "17.0.11",
     "cypress": "9.2.1",
     "eslint-plugin-cypress": "2.12.1",
-    "typescript": "4.5.4"
+    "typescript": "~4.7.4"
```

Comment:
Apparently there is a known issue with type inference for optimistic updates that is resolved with a newer version of typescript.